powa/static/js/components/QueryTooltip.vue

```diff
@@ -30,4 +30,26 @@ const props = defineProps({
 function sqlFormat(value) {
   return hljs.highlightAuto(value, ["pgsql"]).value;
 }
+
+function formatAnchorTag(str) {
+  // If the query is surounded with <a> we catch it to only
+  // format the query
+  var regex = /(<a[^>]*>)([\s\S]*?)(<\/a>)/i;
+  var match = str.match(regex);
+
+  if (match) {
+    var startTag = match[1];
+    var endTag = match[3];
+    var query = match[2];
+    var formattedQuery = sqlFormat(query);
+    return `${startTag}${formattedQuery}${endTag}`;
+  } else {
+    return sqlFormat(str);
+  }
+}
+
+function removeAnchorTags(str) {
+  // In the tooltip we remove anchor tag
+  return str.replace(/<\/?a[^>]*>/g, "");
+}
```

Comment:
I'm not very happy with this. I'd be in favor of avoiding this if possible. We probably should create a `GridCellFormatter`(or the like) component. In my opinion, we should add the `<a />` tag in the Grid. Or if we do, we shouldn't have to remove it afterwards. This seems overly complicated.
Let's think about it a bit more.

Reply:
indeed making a new component will be more readable